### PR TITLE
Adds a height to the li element of gallery

### PR DESCRIPTION
**Pull request by TheDudeWithTheThing**
## BEFORE:

operation_paws_for_homes___our_dogs

## AFTER:

operation_paws_for_homes___our_dogs

### CODE REVIEW

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -140,6 +140,10 @@ form .control-group.error span.error {
   padding-top: 22px;
 }
 
+.gallery li {
+  height: 478px;
```
Comment (TheDudeWithTheThing) [inline app/assets/stylesheets/custom.scss]: 478 is the height of a "normal" gallery box

**Comment by markottaviani** [comment]
Looks good in Chrome, but you wind up with big gaps when viewing on an iPhone.

I can set up with our https://www.browserstack.com login if you don't already have it.   

Might need to go back to the bootstrap grid as per the original dog index: https://github.com/ophrescue/RescueRails/blob/93fc8f66376cd3caf4ebcde3771ff59988c7b198/app/views/dogs/index.html.erb#L27

**Comment by TheDudeWithTheThing** [comment]
Yup that's a pain in the ass though since you have to chunk them 3 at a time, I'll see what I can do.

**Comment by markottaviani** [comment]
Ideally this all goes away after we migrate to newer bootstrap/ Unity / site redesign anyway.  So no worries with it not being perfect.

Mark

**Comment by TheDudeWithTheThing** [comment]
@markottaviani k this should be good now

**Comment by markottaviani** [comment]
👍

**Pull request merged**